opentelemetry-exporter-geneva/geneva-uploader/Cargo.toml

```diff
@@ -4,7 +4,26 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
-opentelemetry-proto = {workspace = true, default-features = false, features = ["logs"]}
+opentelemetry-proto = {workspace = true, default-features = false, features = ["logs", "gen-tonic-messages"]}
+base64 = "0.22"
+serde = { version = "1.0", features = ["derive"] }
+serde_json = { version = "1.0", features = ["raw_value"] }
+uuid = { version = "1.0", features = ["v4"] }
+# TODO - support both native-tls and rustls
+reqwest = { version = "0.12", features = ["native-tls", "native-tls-alpn"]}
```

Comment:
TODO - this would be abstracted with http-client, to remove the indirect dependency on tokio runtime.